app/services/api/charts/sector.rb

```diff
@@ -0,0 +1,148 @@
+module Api
+  module Charts
+    class Sector
+      CP_SCENARIOS = %w[below_2 exact_2 paris not_aligned no_disclosure].freeze
+
+      def initialize(company_scope)
+        @company_scope = company_scope
+      end
+
+      # Returns Companies summaries grouped by their latest MQ assessments Sector's levels.
+      #
+      # Returns data in multiple series.
+      #
+      # @example
+      #   [
+      #    ['1',
+      #      [
+      #        { name: 'Air China', status: 'new' },
+      #        { name: 'China Southern', status: 'new' }
+      #      ]
+      #    ],
+      #    ['3',
+      #      [
+      #        { name: 'Alaska Air', status: 'up' },
+      #        { name: 'IAG', status: 'down' }
+      #      ]
+      #     ]
+      #   ]
+      def companies_summaries
+        @company_scope
+          .includes(:mq_assessments)
+          .group_by { |company| company.mq_assessments.order(:assessment_date).first.level }
```

Comment:
Company has `latest_assessment`, but I think I see a bug there, should use scope ordered by date.

Reply:
also should be renamed to `latest_mq_assessment`

Reply:
or `last_mq_assessment`

Reply:
a306e70 (similar to your fix in https://github.com/Vizzuality/laws_and_pathways/pull/72)

Reply:
:+1: